ada_2_fhir-r4/zibs2020/payload/0.0/zib-NameInformation.xsl

```diff
@@ -74,24 +78,40 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                                 </xsl:otherwise>
                             </xsl:choose>
                         </xsl:attribute>                
-                        <xsl:for-each select="geslachtsnaam/voorvoegsels/@value">
+                        <xsl:for-each select="geslachtsnaam/voorvoegsels">
                             <extension url="http://hl7.org/fhir/StructureDefinition/humanname-own-prefix">
-                                <valueString value="{normalize-space(.)}"/>
+                                <valueString>
+                                    <xsl:call-template name="string-to-string">
+                                        <xsl:with-param name="in" select="."/>
+                                    </xsl:call-template>
+                                </valueString>
                             </extension>
                         </xsl:for-each>
```

Comment:
Ik heb @Ashwin-nictiz dit even laten doen om te oefenen met die templates, maar bij voorvoegsels en voorvoegsels_partner vroeg ik me nog wel af hoe het zit met trailing en leading spaces. String-to-string stript de namelijk wat normaal okee is, maar in het geval van voorvoegsels in de NL context weet ik juist even niet of dat de bedoeling is.